zerver/actions/message_edit.py

```diff
@@ -766,40 +767,46 @@ def user_info(um: UserMessage) -> Dict[str, Any]:
         assert stream_being_edited is not None
         assert topic_name is not None or new_stream is not None
 
+        visibility_policy_to_user_profiles_dict: Dict[int, List[UserProfile]] = defaultdict(list)
```

Comment:
Would `user_profiles_for_visibility_policy` be a better name? Not sure.
In any case I think we can drop the `_dict` suffix. Doesn't seem necessary when we have the `XYZ_to_PQR` style naming :)

Reply:
Yeah, I like that name!

Reply:
Done.